docs/source-app/glossary/secrets.rst

```diff
@@ -0,0 +1,58 @@
+.. _secrets:
+
+#################
+Encrypted Secrets
+#################
+
+Is your App is using data or values (for example: API keys or access credentials) that you don't want to expose in your App code? If the answer is yes, you'll want to use Secrets. Secrets are encrypted values that are stored in the Lightning.ai database and decrypted at runtime.
+
+.. tip::
+	For non-sensitive configuration values, we recommend using :ref:`plain-text Environment Variables <environment_variables>`.
+
+***************
+What did we do?
```

Comment:
I like this as a narrative lol, but we need to keep consistency with our docs.
Titles should be verbs "Add a secret" etc.